paket.dependencies

```diff
@@ -1,24 +1,36 @@
 redirects: on
+storage: none
```

Comment:
Do we need `storage: none`?

Reply:
Paket recommended this for new-style projects in the upgrade guide

Reply:
Note to self: Sounds like I can override this with `storage: packages` on the FAKE dependency to get it working again.

Reply:
Is this the only thing that's blocking us at the moment?